prime-router/src/main/kotlin/Element.kt

```diff
@@ -442,7 +442,7 @@ data class Element(
                 try {
                     // Try to parse using a LocalDate pattern assuming it is in our canonical dateFormatter. Central timezone.
                     val date = LocalDate.parse(formattedValue, dateFormatter)
-                    val zoneOffset = ZoneId.of(USTimeZone.CENTRAL.zoneId).rules.getOffset(Instant.now())
+                    val zoneOffset = ZoneId.of("GMT").rules.getOffset(Instant.now())
```

Comment:
Is there something we can use besides a magic string? Or maybe just move "GMT" up into a constant somewhere.